pyiron_atomistics/lammps/base.py

```diff
@@ -671,6 +674,7 @@ def calc_md(
         langevin=False,
         delta_temp=None,
         delta_press=None,
+        msd=False,
```

Comment:
`calc_vcsgc` is anyhow invoking `calc_md` -- if we do go through with this why *shouldn't* it work for vcasgc?

Reply:
It can for sure. I only need to add the flag in `calc_vcsgc`. For vcsgc, however, the atom species are different, and I thought calculating the msd for all the atoms as a single scalar would not be accurate.

Reply:
Yeah, I don't know to what degree it's physically meaningful, but I have  a mild preference to maintain that the `calc_md` signature is a perfect subset of the `calc_vcsgc` signature.